Sources/ProjectSpec/Project.swift

```diff
@@ -188,12 +184,10 @@ extension Project {
         } else {
             packages = [:]
         }
-        if let localPackages: [String: LocalSwiftPackage] = jsonDictionary.json(atKeyPath: "localPackages") {
-            self.localPackages = localPackages
+        if let localPackages: [String: SwiftPackage] = jsonDictionary.json(atKeyPath: "localPackages") {
```

Comment:
There is no need to support the newer dictionary style from the last PR. We only have to deal with an array of Strings like the currently released version